target/linux/mpc85xx/files/arch/powerpc/boot/dts/br200-wp.dts

```diff
@@ -43,9 +43,7 @@
 		reg = <0 0xffe05000 0 0x1000>;
 		ranges = <0x0 0x0 0x0 0xec000000 0x4000000>;
 
-		nor@0,0 {
-			#address-cells = <1>;
-			#size-cells = <1>;
+		nor@0 {
```

Comment:
Hm, do you have the "warning" that DTC produces on "nor@0,0"? Because nor@0,0 is not per-se wrong, the reg property even has those two "0x0 0x0".

https://elinux.org/Device_Tree_Usage#Memory_Mapped_Devices

These "@0,0" can be valid for a bus.

(As for the #address-size + #size-cells. Yes, these look like remnants from older times and can be removed.

Reply:
yes I do.